mysql-test/suite/rocksdb/t/lock.test

```diff
@@ -200,3 +200,27 @@ DROP TABLE t1, t2;
 # gone so execution of other tests won't be affected by their presence.
 --source include/wait_until_count_sessions.inc
 
+CREATE TABLE t1 (i INT) ENGINE=MyISAM;
+INSERT INTO t1 VALUES (1),(2);
+
+CREATE TABLE t2 (i INT) ENGINE=MyISAM;
+INSERT INTO t2 VALUES (1),(2);
+
+--connect (con1,localhost,root,,test)
+
+--connection default
+HANDLER t2 OPEN h;
+--send
+  CREATE TABLE t3 ENGINE=RocksDB SELECT 1 FROM t1 WHERE SLEEP(0.5) = 0;
```

Comment:
Does the test case rely on this sleep? If so, can we get rid of it, since it could become an unstable test?